pom.xml

```diff
@@ -53,6 +53,7 @@
     <assertj.version>3.9.0</assertj.version>
     <camel.version>2.20.2</camel.version>
     <checkstyle.version>8.5</checkstyle.version>
+    <commons-dbcp2.version>2.2.0</commons-dbcp2.version>
```

Comment:
It could upgrade to use 2.4.0 which includes the fixes of  DBCP-484 and DBCP-491 related to the Narayana

Reply:
OK I'll have to test it first. 2.2.0 is the version used in Spring Boot.